internal/controllers/topology/cluster/patches/variables/variables_test.go

```diff
@@ -416,12 +530,3 @@ func TestMachineDeployment(t *testing.T) {
 func toJSON(value string) apiextensionsv1.JSON {
 	return apiextensionsv1.JSON{Raw: []byte(value)}
 }
-
```

Comment:
Let's keep this func in the test file. I think it's only used in tests

Reply:
Is this about toJsonCompact?

Reply:
Yes.

Reply:
(commented on the left side of the diff view in "files")